docs/content/_navigation.json

```diff
@@ -1116,10 +1120,6 @@
         "title": "Embedded ELT",
         "path": "/integrations/embedded-elt",
         "children": [
-          {
-            "title": "dlt & Dagster",
-            "path": "/integrations/embedded-elt/dlt"
```

Comment:
We will want to add a redirect from `/integrations/embedded-elt/dlt` to `/integrations/dlt`

Reply:
Done in [6cecfbe](https://github.com/dagster-io/dagster/pull/27071/commits/6cecfbee89cc7019877b00f4fe9ca9949c123957)